web/src/pm_list_data.ts

```diff
@@ -81,6 +82,7 @@ export function get_conversations(search_string = ""): DisplayObject[] {
         const recipients_string = people.get_recipients(user_ids_string);
 
         const num_unread = unread.num_unread_for_user_ids_string(user_ids_string);
+        const has_unread_mention = Number(unread.unread_mention_dms.get(user_ids_string)?.size) > 0;
```

Comment:
This is an abstractions violation, reaching into the data object and directly manipulating it; the `unread` module should expose a `unread.get_unread_dm_mention_count` function that these can call.